frontend/src/cards/MapCard.tsx

```diff
@@ -125,6 +130,17 @@ function MapCardWithKey(props: MapCardProps) {
             (row: Row) => row[activeBreakdownVar] === activeBreakdownFilter
           );
 
+        const highestFive = dataForActiveBreakdownFilter
+          .sort((rowA: Row, rowB: Row) =>
+            rowA[metricConfig.metricId] <= rowB[metricConfig.metricId] ? 1 : -1
+          )
+          .slice(0, 5);
```

Comment:
Can we make the 5 into a constant at the top, and then also change the variable names here to mostAffectedGeos or something like that?

Reply:
Why the variable name change? What about highestRates and lowestRates ? I think mostAffectedGeos is less clear if you think about other metrics like "uninsured individuals" i think highest rates of uninsured is more straightforward than most affected geos for uninsured ?

Reply:
Suggesting the variable name change as the 5 will become a constant which can then be updated independently

Good point, highestRates/lowestRates sgtm

Reply:
ah gotcha word

Reply:
Oh, I see now that the text "Top 5 Highest Rates" has a hard-coded 5 as well, we should use the constant there as well if possible - if it's not possible, I'm fine with leaving everything as-is (ie leave the 5 hard coded everywhere)